src/main/scala/centaur/test/CheckFiles.scala

```diff
@@ -0,0 +1,82 @@
+package centaur.test
+
+import com.google.cloud.storage.{Storage, Blob, BlobId}
+import com.google.cloud.storage.Storage.BlobListOption
+import scala.collection.JavaConverters._
+import java.io.File
+
+sealed trait CheckFiles {
+  def checkDirectorySize: String => Int 
+  def deleteExistingFiles: String => Unit
+}
+
+case class JesCheckFiles(storage: Storage) extends CheckFiles {
+  import GCS._
+  import storage._
+
+  def checkDirectorySize: String => Int = 
```

Comment:
I imagine there's a small overhead with a def since it probably gets re-evaluated every time it's used. But that's total nitpicking don't bother changing it I was just curious